src/main/java/uk/gov/companieshouse/api/strikeoffobjections/file/FileTransferApiClient.java

```diff
@@ -27,7 +29,12 @@
     private static final String UPLOAD = "upload";
     private static final String CONTENT_DISPOSITION_VALUE = "form-data; name=%s; filename=%s";
     private static final String NULL_RESPONSE_MESSAGE = "null response from file transfer api url";
+    private static final String GET_URI = "%s/%s";
```

Comment:
this and delete uri are the same value, would it be possible to merge the constants?

Reply:
We could but it is more readable this way and if one were to change.. so I've kep them separate.